DataConnectors/Okta Single Sign-On/AzureFunctionOktaSSO_V2/AzureFunctionOktaSSO/run.ps1

```diff
@@ -52,13 +52,20 @@ $TotalRecordCount = 0
 $apiToken = $env:apiToken
 $uri = $env:uri
 $StartDate = [System.DateTime]::UtcNow.ToString("yyyy-MM-ddT00:00:00.000Z") # set default fallback start time to 0:00 UTC today
+$logAnalyticsUri = $env:logAnalyticsUri
 
 # Define the Log Analytics Workspace ID and Key and Custom Table Name
 $customerId = $env:workspaceId
 $sharedKey =  $env:workspaceKey
 $LogType = "Okta"
 $TimeStampField = "published"
 
+# Returning if the Log Analytics Uri is in incorrect format.
+# Sample format supported: https://" + $customerId + ".ods.opinsights.azure.com
+if($logAnalyticsUri -notmatch 'https:\/\/([\w\-]+)\.ods\.opinsights\.azure.([\w\.]+)')
+{
+    Write-Error -Message "OKTASSO: Invalid Log Analytics Uri." -ErrorAction Stop
```

Comment:
Why write-error? this has to be an exception and terminate the execution, right?

https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/write-error?view=powershell-7.1

Reply:
updated.